### PR TITLE
did some TODO's

### PR DESCRIPTION
**Pull request by T14Raptor**

### CODE REVIEW

File: plugin-manifest-spec.md [hunk 1]
```diff
@@ -98,10 +98,23 @@ List of Minecraft PE network protocol versions the plugin is compatible with. Pl
...
 Definitions of commands implemented by this plugin in the `onCommand()` of the `PluginBase`.
-TODO: examples
+`Example:`
+```
+commands:
+ example:
+  description: Example command
+  usage: "/example"
+  permission: exampleperm.command.example
+```
```
Comment (dktapps) [inline plugin-manifest-spec.md]: Perhaps include an example `permission-message` and `aliases` usage here.

File: plugin-manifest-spec.md [hunk 2]
```diff
@@ -98,10 +98,23 @@ List of Minecraft PE network protocol versions the plugin is compatible with. Pl
...
 
 Tree of permissions defined by this plugin, usually used for commands.
-TODO: examples
\ No newline at end of file
+`Example: `
+```
+permissions:
+ exampleperm.command.example:
+  description: "Allows the user to run the example command"
+  default: true
```
Comment (dktapps) [inline plugin-manifest-spec.md]: We could use an example of the `children` usage here. Also, maybe add a comment with the possible values for the `default`.

File: plugin-manifest-spec.md [hunk 3]
```diff
@@ -98,10 +98,23 @@ List of Minecraft PE network protocol versions the plugin is compatible with. Pl
...
+  permission: exampleperm.command.example
+```
 
 ### `permissions`
 Type: array
 
 Tree of permissions defined by this plugin, usually used for commands.
-TODO: examples
\ No newline at end of file
+`Example: `
```
Comment (dktapps) [inline plugin-manifest-spec.md]: why is this in grave quotes?

File: plugin-manifest-spec.md [hunk 4]
```diff
@@ -98,10 +98,23 @@ List of Minecraft PE network protocol versions the plugin is compatible with. Pl
 Type: array
 
 Definitions of commands implemented by this plugin in the `onCommand()` of the `PluginBase`.
-TODO: examples
+`Example:`
```
Comment (dktapps) [inline plugin-manifest-spec.md]: why is this in grave quotes?

File: plugin-manifest-spec.md [hunk 5]
```diff
@@ -98,10 +98,23 @@ List of Minecraft PE network protocol versions the plugin is compatible with. Pl
...
+```
```
Comment (dktapps) [inline plugin-manifest-spec.md]:
```suggestion
```yml
```

File: plugin-manifest-spec.md [hunk 6]
```diff
@@ -98,10 +98,23 @@ List of Minecraft PE network protocol versions the plugin is compatible with. Pl
...
 
 ### `permissions`
 Type: array
 
 Tree of permissions defined by this plugin, usually used for commands.
-TODO: examples
\ No newline at end of file
+`Example: `
+```
```
Comment (dktapps) [inline plugin-manifest-spec.md]:
```suggestion
```yml
```

File: plugin-manifest-spec.md (hunk 1 above)
Comment (T14Raptor) [reply plugin-manifest-spec.md]: kk bb, and permission-message?

File: plugin-manifest-spec.md (hunk 2 above)
Comment (T14Raptor) [reply plugin-manifest-spec.md]: ye i was just aboot to do that!

File: plugin-manifest-spec.md (hunk 4 above)
Comment (T14Raptor) [reply plugin-manifest-spec.md]: cuz it wants too

File: plugin-manifest-spec.md (hunk 6 above)
Comment (T14Raptor) [reply plugin-manifest-spec.md]: sry dooder, was in a hurry when makin this, i am updating everyting

**Comment by T14Raptor** [comment]
luv u too jack <3 k im done

File: plugin-manifest-spec.md [hunk 7]
```diff
@@ -98,10 +98,35 @@ List of Minecraft PE network protocol versions the plugin is compatible with. Pl
...
 
 Tree of permissions defined by this plugin, usually used for commands.
-TODO: examples
\ No newline at end of file
+Example: 
+```yml
+permissions:
+ exampleperm:
+  description: "Allows the user to do all example things"
+   default: true # can also be false, or op
```
Comment (dktapps) [inline plugin-manifest-spec.md]: Indentation is one space too far in for every line after this one (including this).

File: plugin-manifest-spec.md [hunk 8]
```diff
@@ -98,10 +98,35 @@ List of Minecraft PE network protocol versions the plugin is compatible with. Pl
...
+   children:
+     exampleperm.command:
```
Comment (dktapps) [inline plugin-manifest-spec.md]: Inconsistent indentation. One space or two spaces is fine, but don't mix it.

File: plugin-manifest-spec.md [hunk 9]
```diff
@@ -98,10 +98,35 @@ List of Minecraft PE network protocol versions the plugin is compatible with. Pl
...
+      description: "Allows the user to run all example commands"
+      default: true # can also be false, or op
+      children:
+        exampleperm.command.example:
```
Comment (dktapps) [inline plugin-manifest-spec.md]: Inconsistent indentation. One space or two spaces is fine, but don't mix it.

File: plugin-manifest-spec.md (hunk 7 above)
Comment (T14Raptor) [reply plugin-manifest-spec.md]: luv u too

File: plugin-manifest-spec.md (hunk 9 above)
Comment (T14Raptor) [reply plugin-manifest-spec.md]: stop spamming the same stuff
bad boi

**Pull request merged**